site/docs/tools/iceberg_client.md

```diff
@@ -1,23 +1,18 @@
-# Spark
-
-Nessie can be used with Spark in different ways. These include:
-
-## Iceberg client
+# Iceberg Client
 
 Nessie works seamlessly with Iceberg in Spark2 and Spark3. Nessie is implemented as a custom iceberg
 [catalog](http://iceberg.apache.org/custom-catalog/) and therefore supports all features available to any Iceberg
-client. This includes Spark structured streaming, Presto and Flink. See the [iceberg docs](https://iceberg.apache.org)
+client. This includes Spark structured streaming, Presto, Flink and Hive. See the [iceberg docs](https://iceberg.apache.org)
 for more info.
 
 !!! note
-    You can follow along an interactive demo in a [Jupyter Notebook via Google Colab](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/colab/nessie-iceberg-demo-nba.ipynb)
-    or in a [Jupyter Notebook via Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-demo-nba.ipynb).
+    You can follow along an interactive demo in a [Jupyter Notebook via Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-demo-nba.ipynb).
     
-    Detailed steps on how to set up Pyspark + Iceberg + Nessie with Python is available on
-    [Colab](https://colab.research.google.com/github/projectnessie/nessie-demos/blob/main/colab/nessie-iceberg-spark-setup.ipynb)
-    and [Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-spark-setup.ipynb)
+    Detailed steps on how to set up Pyspark + Iceberg + Nessie with Python is available on [Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=colab%2Fnessie-iceberg-spark-setup.ipynb)
 
 
+## Spark
```

Comment:
Should we keep `spark.md` as `iceberg_spark.md` as a separate file?
And list all the Spark demos prominently at the top?

Reply:
Actually I am thinking that every integration page gets its own demo link, perhaps it might be more clear from the user perspective about the demo with its corresponding documentation page

Reply:
I'm not sure about this `Spark` section header here :thinking: it looks kind of odd to me. My undertanding is that Nessie integrates with Spark by means of an Iceberg catalog. Having a separate `Spark` header in the middle of the page makes it look like this is a separate use case.... looks confusing to me. May I'm missing something :thinking:

Reply:
Yeah I understand what you mean. To mitigate this issue, I will be pushing [this](https://github.com/projectnessie/nessie/pull/1965#issuecomment-919000380) later today, so it will make Spark as its own page(s), one under Iceberg section and one under Data Lake section. IMHO as a newbie, with this structure at least I am getting more clarity how the relation between Spark,Iceberg and Nessie play all together.